includes/class-wp-job-manager-shortcodes.php

```diff
@@ -521,7 +525,7 @@ public function output_job( $atts ) {
 		);
 
 		if ( ! $atts['id'] ) {
-			return;
+			return null;
```

Comment:
This was to get rid of a sniff issue. It wasn't a bug, but it doesn't like `void` being used for a string.